docs/components/interactive-map.md

```diff
@@ -0,0 +1,121 @@
+# InteractiveMap
+
+A stateful `React` component.
```

Comment:
Not sure about the "stateful" comment here at the top, it is very technical, in what sense will it help a user coming here to see this as the first statement? If we have a `Remarks` section at the end of this page, and we need to say this, that is where I think it should go. Or at least at the bottom of the introduction.